dolphinscheduler-task-plugin/dolphinscheduler-task-sql/pom.xml

```diff
@@ -55,5 +55,10 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.dolphinscheduler</groupId>
+            <artifactId>dolphinscheduler-common</artifactId>
+        </dependency>
+
```

Comment:
Please don't add `dolphinscheduler-common` into task plugin module.

Reply:
I need to use the LoggerUtils.buildTaskId() method,Can I write the same method in the dolphinscheduler-task-api